CI-Build: Breaks due to `motore` update

> error: package `motore-macros v0.4.1` cannot be built because it requires rustc 1.77.0 or newer, while the currently active rustc version is 1.75.0
> Either upgrade to rustc 1.77.0 or newer, or use
> cargo update motore-macros@0.4.1 --precise ver
> where `ver` is the latest version of `motore-macros` supporting rustc 1.75.0
> Error: Process completed with exit code 101.

...I think this comes from the transitive dependency defined by "[volo-thrift](https://crates.io/crates/volo-thrift/0.9.2/dependencies)", which specifies `motore = ^0.4` ...motore had an update 4 hours ago to ver 0.4.1 which seems to break the CI.

I'm not sure what's the best approach here. Upgrade the rust-version? Running `cargo update motore-macros@0.4.0 --precise 0.4.0` fixes the build  locally.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ homepage = "https://rust.iceberg.apache.org/"
 
 repository = "https://github.com/apache/iceberg-rust"
 license = "Apache-2.0"
-rust-version = "1.75.0"
+rust-version = "1.77.1"
 
 [workspace.dependencies]
 anyhow = "1.0.72"

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -16,5 +16,5 @@
 # under the License.
 
 [toolchain]
-channel = "1.75.0"
+channel = "1.77.1"
 components = ["rustfmt", "clippy"]

